src/main/java/apoc/meta/Meta.java

```diff
@@ -101,6 +101,9 @@ public static Types of(Class<?> type) {
         }
 
         public static Types from(String typeName) {
+            if (typeName == null) {
```

Comment:
In current APOC if user specifies a type mapping of null, they get an NPE.

The column doesn't even have to exist in the input. 

Reproduction: 
```
CALL apoc.load.csv("https://raw.githubusercontent.com/moxious/cypher-surface/master/surface-3.4.0.csv", { mapping: { whateverColumn: { type: null }}});
```